app/code/Magento/Catalog/Pricing/Price/CustomOptionPrice.php

```diff
@@ -161,6 +163,19 @@ public function getCustomOptionRange($getMin, $priceCode = \Magento\Catalog\Pric
         return $this->priceCurrency->convertAndRound($optionValue);
     }
 
+    /**
+     * Get custom amount without currency convert
+     *
+     * @param float|string $amount
+     * @param null|array $context
+     * @return AmountInterface
+     */
+    public function getCustomRoundAmount($amount, $context = [])
```

Comment:
There is no getCustomRoundAmount in CustomOptionPriceInterface. Please consider to find other solution.